packages/flutter/lib/src/widgets/layout_builder.dart

```diff
@@ -187,18 +203,20 @@ class _LayoutBuilderElement<ConstraintType extends Constraints> extends RenderOb
     super.unmount();
   }
 
-  // The constraints that were passed to this class last time it was laid out.
-  // These constraints are compared to the new constraints to determine whether
-  // [ConstrainedLayoutBuilder.builder] needs to be called.
-  ConstraintType? _previousConstraints;
+  // The LayoutInfo that were used to invoke the layout callback with last time,
+  // during layout. These LayoutInfo is compared to the new ones to determine
+  // whether [LayoutBuilderBase.builder] needs to be called.
```

Comment:
nit: I am never sure about this, but I think "info" or "information" is a singular word, so it should be e.g. "was" instead of "were" etc.